### ISSUE TITLE
Vivox Voice Morphing not enabling

### ISSUE THREAD

**Issue by WendiLinden** [secondlife/viewer#1888]
### Environment

Second Life Release 7.1.8.9375512768 (64bit)
Release Notes

CPU: 12th Gen Intel(R) Core(TM) i9-12900H (2918.41 MHz)
Memory: 32459 MB
OS Version: Microsoft Windows 10/11 64-bit (Build 22621.3737)
Graphics Card Vendor: NVIDIA Corporation
Graphics Card: NVIDIA GeForce RTX 3060 Laptop GPU/PCIe/SSE2

Windows Graphics Driver Version: 30.0.15.1215
OpenGL Version: 4.6.0 NVIDIA 512.15

Window size: 1888x920
Font Size Adjustment: 96pt
UI Scaling: 1
Draw distance: 128m
Bandwidth: 3000kbit/s
LOD factor: 1.5
Render quality: 4
Texture memory: 5255MB
Disk cache: Max size 1638.4 MB (100.0% used)

J2C Decoder Version: KDU v7.10.4
Audio Driver Version: FMOD Studio 2.02.20
Dullahan: 1.14.0.202310131404
  CEF: 118.4.1+g3dd6078+chromium-118.0.5993.54
  Chromium: 118.0.5993.54
LibVLC Version: 3.0.16
Voice Server Version: Not Connected
June 28 2024 18:32:32

### Description

Confirmed a Resident-reported  bug with Vivox Voice Morphing on the [Premium Morph Services] island. Clicking for voice morphing is not enabling voice morphs in the viewer. I presume this may have started with their recent update on 27th June @ 7am PT/SLT

It is unclear, but it is likely that paid voice morphs are similarly not working. 

### Reproduction steps

1. Go to [Premium Morph Services] (Notye: this will only be accessible with Premium or Premium Plus accounts, or, presumably, Linden and Comp accounts)

2. Click on any of the kiosks (I would recommend MegaMorph). The following text will appear

_[19:35] MegaMorphs: Your request is being processed...
[19:35] MegaMorphs: Thank you for subscribing to Second Life Voice Morphing! Click secondlife:///app/voice/effects/refresh to activate the new voice morphs in your viewer._

3. Clicking the link does nothing. The expected "New Voice Morphs are Available" toast does not appear. 

4. Checking Communicate > Voice Morphing does not show any of the expected voice morphs (note that this should be tested with an account that doesn't already have them from before 27 June)

**Comment by maestrolinden** [comment]
I followed the repro steps in Second Life Release 7.1.8.9375512768 (64bit), with the 'VoiceFonts' log tag set to DEBUG level, using an account that didn't previously have voice morph fonts available (agent_id 70243b06-a65e-4e2b-b419-4844fbc3ea11 ; Communicate -> Voice Morphing didn't show any voices).  

Upon clicking  secondlife:///app/voice/effects/refresh and observing the lack of "new voice morphs are available!" toast, my viewer logs this:

```
2024-07-01T18:44:39Z INFO # newview/llurldispatcher.cpp(162) dispatchApp : cmd: voice path: ['effects','refresh'] query: 
2024-07-01T18:44:39Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6961) LLVivoxVoiceClient::accountGetSessionFontsSendMessage : Requesting voice font list.
2024-07-01T18:44:39Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6979) LLVivoxVoiceClient::accountGetTemplateFontsSendMessage : Requesting voice font template list.
2024-07-01T18:44:40Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(7131) LLVivoxVoiceClient::notifyVoiceFontObservers : Notifying voice effect observers. Lists changed: 0
2024-07-01T18:44:40Z INFO # newview/llurldispatcher.cpp(162) dispatchApp : cmd: voice path: ['effects','refresh'] query: 
2024-07-01T18:44:40Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6961) LLVivoxVoiceClient::accountGetSessionFontsSendMessage : Requesting voice font list.
2024-07-01T18:44:40Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6979) LLVivoxVoiceClient::accountGetTemplateFontsSendMessage : Requesting voice font template list.
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 6d124281-63be-812f-c046-750c19a30086 (3) Nightmare
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 2ec271c9-b7d9-9fc1-ec57-9d5434c78d2d (5) Creepy
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 63463a60-c06f-d4b1-1ed7-ce28ccd70e35 (11) Nano
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 8b9bcec3-bca7-6e29-d06d-3ef740b2c51d (13) Beast
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z aa387a6e-6149-613d-f80b-9355a8c2e9d3 (15) Stallion
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 1608934d-7bdd-0909-1fcc-375912d00626 (17) Cyber
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 51a1f46e-f14a-38a8-df23-58c5cbea5497 (19) Husky
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 7a370909-d756-7e7b-98f8-fbaf97cd56a6 (25) Sultry
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 4de25c7e-a362-9278-8c7b-4211b636d423 (63) Radio
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z f28fe914-56f9-9a2b-7a4d-8d80757af392 (65) Buff
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z e19f0ebc-48e4-2bda-324d-037939f8ce5f (67) Intercom
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 2312600d-d07d-3c93-a254-d28c8010e5ee (69) Demon
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z b4dd4d63-a7ac-a43e-bfd2-8d1665ca5041 (71) Shorty
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 0585a82f-6698-fe68-5bea-4a87865be376 (73) Foxy
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 1dae461a-ca0b-0d83-03da-69486cbe6029 (75) Flirty
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 16bdc5c0-c19b-6598-8bb7-6219473e83f4 (77) Helium
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z a91392bc-ea06-ce70-485d-ca172fb1cb68 (81) Arena
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 64784194-b439-01f6-f19f-5ad7af6b94cb (125) Mini
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 9bc0f416-a279-0766-47ed-90c48c8f34ce (131) Micro
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z f5f08739-15d4-cd93-8ba2-4b1ceee73b88 (139) Macho
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z ceb14108-1726-0c05-bf99-37508cc4645e (141) Vixen
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 98b4cd22-15c5-1750-036a-431cf131cffd (147) Sexy
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 100ce832-797a-e8af-efd8-5a5534772a8e (163) Sneaky
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 5e8fc8dc-ec13-4587-395f-37cebda07d61 (177) Buzz
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 29352dae-784c-7cc8-bdac-293bb1353887 (181) Thunder
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z dcd0788a-9198-c999-57e8-31555ef3dded (187) Roxanne
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 690c9f65-b3d1-5f70-e7ca-189dd07b367c (189) Camille
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 732841b0-011c-3d59-14cd-51b0547fd867 (191) Rachel
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 39252c0d-af08-b82e-faf8-5d50eef99d14 (193) Samantha
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 564451fc-9ac4-c53d-8f05-7def7700107b (195) Sabrina
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z e32f9028-bdd7-849c-e449-654adb217451 (197) PopBot
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 54897258-8f0e-0971-2bae-4e75521855af (199) Robot
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 5458c2ec-0efa-155d-611f-37386502aa57 (201) DeepBot
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z fa0f092c-312d-40a4-fa69-66bfa6a76f42 (203) WhinyBot
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 778b9852-e9f5-e3ff-6c05-777c3fcd2260 (205) CreepyBot
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 824693d1-345a-5909-e1a6-c105bb472e2c (207) Halloween_2010_Bonus
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(7131) LLVivoxVoiceClient::notifyVoiceFontObservers : Notifying voice effect observers. Lists changed: 1
2024-07-01T18:44:41Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(7131) LLVivoxVoiceClient::notifyVoiceFontObservers : Notifying voice effect observers. Lists changed: 0
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 6d124281-63be-812f-c046-750c19a30086 (3) Nightmare
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 2ec271c9-b7d9-9fc1-ec57-9d5434c78d2d (5) Creepy
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 63463a60-c06f-d4b1-1ed7-ce28ccd70e35 (11) Nano
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 8b9bcec3-bca7-6e29-d06d-3ef740b2c51d (13) Beast
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z aa387a6e-6149-613d-f80b-9355a8c2e9d3 (15) Stallion
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 1608934d-7bdd-0909-1fcc-375912d00626 (17) Cyber
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 51a1f46e-f14a-38a8-df23-58c5cbea5497 (19) Husky
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 7a370909-d756-7e7b-98f8-fbaf97cd56a6 (25) Sultry
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 4de25c7e-a362-9278-8c7b-4211b636d423 (63) Radio
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z f28fe914-56f9-9a2b-7a4d-8d80757af392 (65) Buff
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z e19f0ebc-48e4-2bda-324d-037939f8ce5f (67) Intercom
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 2312600d-d07d-3c93-a254-d28c8010e5ee (69) Demon
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z b4dd4d63-a7ac-a43e-bfd2-8d1665ca5041 (71) Shorty
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 0585a82f-6698-fe68-5bea-4a87865be376 (73) Foxy
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 1dae461a-ca0b-0d83-03da-69486cbe6029 (75) Flirty
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 16bdc5c0-c19b-6598-8bb7-6219473e83f4 (77) Helium
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z a91392bc-ea06-ce70-485d-ca172fb1cb68 (81) Arena
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 64784194-b439-01f6-f19f-5ad7af6b94cb (125) Mini
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 9bc0f416-a279-0766-47ed-90c48c8f34ce (131) Micro
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z f5f08739-15d4-cd93-8ba2-4b1ceee73b88 (139) Macho
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z ceb14108-1726-0c05-bf99-37508cc4645e (141) Vixen
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 98b4cd22-15c5-1750-036a-431cf131cffd (147) Sexy
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 100ce832-797a-e8af-efd8-5a5534772a8e (163) Sneaky
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 5e8fc8dc-ec13-4587-395f-37cebda07d61 (177) Buzz
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 29352dae-784c-7cc8-bdac-293bb1353887 (181) Thunder
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z dcd0788a-9198-c999-57e8-31555ef3dded (187) Roxanne
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 690c9f65-b3d1-5f70-e7ca-189dd07b367c (189) Camille
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 732841b0-011c-3d59-14cd-51b0547fd867 (191) Rachel
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 39252c0d-af08-b82e-faf8-5d50eef99d14 (193) Samantha
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 564451fc-9ac4-c53d-8f05-7def7700107b (195) Sabrina
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z e32f9028-bdd7-849c-e449-654adb217451 (197) PopBot
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 54897258-8f0e-0971-2bae-4e75521855af (199) Robot
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 5458c2ec-0efa-155d-611f-37386502aa57 (201) DeepBot
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z fa0f092c-312d-40a4-fa69-66bfa6a76f42 (203) WhinyBot
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 778b9852-e9f5-e3ff-6c05-777c3fcd2260 (205) CreepyBot
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(6766) LLVivoxVoiceClient::addVoiceFont : Template 1970-01-01T00:00:00Z 824693d1-345a-5909-e1a6-c105bb472e2c (207) Halloween_2010_Bonus
2024-07-01T18:44:43Z DEBUG #VoiceFont# newview/llvoicevivox.cpp(7131) LLVivoxVoiceClient::notifyVoiceFontObservers : Notifying voice effect observers. Lists changed: 1
```

Despite the voices listed in the `LLVivoxVoiceClient::addVoiceFont` messages, 'Communicate -> Voice Morphing' remained unchanged in my viewer.   The 1970 timestamps shown for each voice font template seems suspicious to me (a parsing error?) but I'm unfamiliar with what the logs look like when morphs are added successfully.

**Comment by maestrolinden** [comment]
This issue has been resolved, following a fix from Vivox.  I was able to confirm that account with no previous voice morph usage was able to obtain a few different voice morphs from the 'Premium Morph Services' location.
